### ISSUE TITLE
Google Pay throws an error when the "button" parameter is present in the configuration

### ISSUE THREAD

**Issue by mmartinov1c** [braintree/braintree-web-drop-in#701]
### General information

* SDK/Library version: 1.26.1
* Environment: Sandbox
* Browser and OS: Chromium 88.0.4324.182 on Linux/Ubuntu

### Issue description
Google Pay does not work when the "button" parameter is present in the configuration.

Reference for button option: https://braintree.github.io/braintree-web-drop-in/docs/current/module-braintree-web-drop-in.html#~googlePayCreateOptions

My config:

 ```
dropin.create(
                {
                    authorization: mytoken,
                    container: '#braintree-container',
                    threeDSecure: true,
                    dataCollector: true,
                    googlePay: {
                        googlePayVersion: 2,
                        merchantId: 'my-google-merchant-id', 
                        transactionInfo:{
                            totalPriceStatus: 'FINAL',
                            totalPrice: '123.45',
                            currencyCode: 'EUR',
                        },
                        // shippingOptionRequired: false,
                        // changes color but throws an exception
                        button: {
                           buttonColor: 'white',
                        },
            	   },
                   
                },
                (error, dropinInstance) => {
                    if (error) {
                        console.error(error);
                        return;
                    }
                    // ...
                }
);
 ```
            
Google displayed error: **Unexpected developer error, please try again later.**

Dropin displayed error: **Developer Error: Something went wrong. Check the console for details.**

Console: **pay.js:101 DEVELOPER_ERROR in loadPaymentData: PaymentDataRequest.button does not exist.**

Console stack trace:

```
initFeatureFlags @ runtime-core.esm-bundler.js:3413
baseCreateRenderer @ runtime-core.esm-bundler.js:3672
createRenderer @ runtime-core.esm-bundler.js:3660
ensureRenderer @ runtime-dom.esm-bundler.js:1188
createApp @ runtime-dom.esm-bundler.js:1205
./resources/js/bootstrap.js @ bootstrap.js:43
__webpack_require__ @ bootstrap:18
./resources/js/app.js @ app.js:1
__webpack_require__ @ bootstrap:18
checkDeferredModulesImpl @ jsonp chunk loading:110
webpackJsonpCallback @ jsonp chunk loading:92
(anonymous) @ app.js?id=e78f810ea5fe8d16d195:1
Booking.vue:245 Proxy
PersonalInfoAndPayment.vue:619 Object
pay.js:101 DEVELOPER_ERROR in loadPaymentData: PaymentDataRequest.button does not exist.
B @ pay.js:101
(anonymous) @ pay.js:242
Promise.then (async)
k.Sb @ pay.js:242
Sc @ pay.js:146
k.V @ pay.js:144
k.V @ pay.js:235
(anonymous) @ pay.js:249
k.V @ pay.js:249
GooglePayView.tokenize @ dropin.js:22175
onClick @ dropin.js:22141
dropin.js:22186 Error
    at pay.js:242
(anonymous) @ dropin.js:22186
Promise.catch (async)
GooglePayView.tokenize @ dropin.js:22182
onClick @ dropin.js:22141
```

**Comment by crookedneighbor** [comment]
Ahh, we need to remove that option when actually creating the payment request. I think Google used to ignore any params that weren't valid for the payment request, but now validates them.

**Comment by crookedneighbor** [comment]
The fixed was merged to master, but is not yet released. Going to re-open this until the release actually goes out.

**Comment by jplukarski** [comment]
This should be fixed in version 1.27.0
